README.md

```diff
@@ -49,6 +49,42 @@ Run `sh build.sh help` to see all the actions you can perform (build ios/osx, ge
 
 Executing the examples under the `examples/` folder, requires that you have built the `Realm.framework`.
 
+
+## Branches
+
+Here is an overview of some long-running branches we're currently using in development, which you may wish to use.
+
+_:warning: Feedback is very welcome for unreleased version, but please aware that we can't provide support to the same
```

Comment:
Is this really necessary? Can we wait until we see what kind of requests we get?